direct-auth/OktaDirectAuth/OktaDirectAuth/Onboarding/View Controllers/AuthenticateViewController.swift

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2021, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import UIKit
+import OktaIdxAuth
+
+@IBDesignable
+class AuthenticateViewController: UIViewController, SigninController {
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var usernameField: UITextField!
+    @IBOutlet weak var passwordField: UITextField!
+    @IBOutlet weak var nextButton: UIButton!
+
+    var auth: OktaIdxAuth?
+
+    @IBAction func cancelAction(_ sender: Any) {
```

Comment:
Can the methods `cancelAction`, `nextAction`, `handle`, `showChangePassword` be private?

The reason why I'm asking, if I was a new developer in Okta, I would try to understand how things work via samples. 
If I don't see `private`, `private(set)`, `final` it makes me feel that the things are overridden or accessed externally. It takes time to understand whether it's true or not.

Reply:
They can be, they are left this way since it's the default visibility that Xcode establishes when creating outlets, and I didn't want to deviate too much from the basics.  Especially if a developer wants to tweak these going forward.